.travis.yml

```diff
@@ -8,17 +8,15 @@ addons:
 rvm:
   - 2.5.3 # deployed
 
-jdk:
-  - oraclejdk8
-
 email: false
 
 before_install:
-  - docker pull suldlss/workflow-server:latest
-  - docker run -d -p 127.0.0.1:3001:3000 suldlss/workflow-server:latest
-  - docker run -d -p 8984:8983 -v $PWD/solr_conf/conf/:/myconfig solr:7 solr-create -c argo -d /myconfig
-  - docker run -d -p 8983:8080 suldlss/fcrepo:no-messaging-latest
-  - docker ps -a
+  - sudo rm /usr/local/bin/docker-compose
```

Comment:
Why do we need to remove the system version?  Can you add a comment to explain the reason?

Reply:
Comment added.